andela/mrm_api
# #162151559 Return a descriptive error message for invalid tokens

#### What does this PR do?
- Add an error message when a user has an invalid token

#### How should this be manually tested?
- Run server `http://127.0.0.1:5000/mrm`
- Make any analytics query with an expired token
- It should return an error message stating that the token is invalid

#### Any background context you want to provide?
- In case you do not have access to an invalid token, here is one:

> eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJVc2VySW5mbyI6eyJpZCI6Ii1MOW8yZFI4Q3dFbGNVdEdmWF9IIiwiZmlyc3RfbmFtZSI6IlRhcGl3YSIsImxhc3RfbmFtZSI6Ik1hcGZ1bmRlbWF0c3ZhIiwiZmlyc3ROYW1lIjoiVGFwaXdhIiwibGFzdE5hbWUiOiJNYXBmdW5kZW1hdHN2YSIsImVtYWlsIjoidGFwaXdhLm1hcGZ1bmRlbWF0c3ZhQGFuZGVsYS5jb20iLCJuYW1lIjoiVGFwaXdhIE1hcGZ1bmRlbWF0c3ZhIiwicGljdHVyZSI6Imh0dHBzOi8vbGg2Lmdvb2dsZXVzZXJjb250ZW50LmNvbS8tZFVQS05uLTBhZ0EvQUFBQUFBQUFBQUkvQUFBQUFBQUFBQUEveTJKcmlTU1BOVncvcGhvdG8uanBnP3N6PTUwIiwicm9sZXMiOnsiRmVsbG93IjoiLUtYR3kxRUIxb2ltalFnRmltNkMiLCJBbmRlbGFuIjoiLUtpaWhmWm9zZVFlcUM2YldUYXUifX0sImlhdCI6MTU0MDE5MzI0MiwiZXhwIjoxNTQyNzg1MjQyLCJhdWQiOiJhbmRlbGEuY29tIiwiaXNzIjoiYWNjb3VudHMuYW5kZWxhLmNvbSJ9.iWoYZcHHdwE4VwoW5ggFLgmdrRec_Mdpj0NBVoN3rAVACT6InCemhBBEtnP-NvIaWMmxAmhouaccfgMi1A_tvGFrLBPI9D2lHNIhZuD3tCCfjdjqajY2j82D3wqacgqtuTEskfn7CZvp4Lw3nzfrOZ2X_6cuQXusy-_UaY1rQ_g

#### What are the relevant pivotal tracker stories?
- [#162151559](https://www.pivotaltracker.com/story/show/162151559)

#### Screenshots 
**Error message on GraphQL endpoint**
<img width="1101" alt="screenshot 2018-11-22 at 11 35 29" src="https://user-images.githubusercontent.com/33119403/48917798-bb06eb80-ee99-11e8-9f88-85219ec5eca0.png">

**Error message on REST endpoint**
<img width="1039" alt="screenshot 2018-11-22 at 11 35 37" src="https://user-images.githubusercontent.com/33119403/48917799-bb9f8200-ee99-11e8-846a-c2e3c9961d15.png">
